redisinsight/api/src/modules/auth/window-auth/window.auth.strategy.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface IWindowAuthStrategyData {
+  isExists: boolean
+}
+export interface IWindowAuthStrategy {
```

Comment:
I think we don't need this interface since we will you abstract class

Reply:
removed